python/paddle/fluid/layers/control_flow.py

```diff
@@ -912,6 +913,93 @@ def _complete(self):
                    "is_test": self.is_test})
 
 
+def while_loop(cond, body, loop_vars, name=None):
+    """
+    while_loop is one of the control flows. Repeats while_loop `body` until `cond` returns False.
+
+    Args:
+        cond(Callable): A callable returning a boolean tensor controlling whether to continue looping.
+        body(Callable): A callable returning a tuple or list of tensors of the same arity(length and structure)
```

Comment:
Just minor English knowledge, you should write "arity (length and structure)" in English text. Unlike code, you can put "(" after function/variable, normal English parentheses have space before them.

Reply:
Done.